Source/Microsoft.Teams.Apps.CompanyCommunicator/appsettings.json

```diff
@@ -18,7 +18,7 @@
   "MicrosoftAppPassword": "[MicrosoftAppPassword]",
   "ServiceBusConnection": "[Service Bus Connection]",
   "StorageAccountConnectionString": "[Storage Account Connection String]",
-  "DisableAuthentication": "false",
+  "DisableMustBeValidUpnCheck": "false",
```

Comment:
DisableMustBeValidUpnCheck does not in fact disable authentication, right? It still must have a valid token, it just doens't have to be in the list? Or does it turn of authentication altogether?